shell.core/src/main/java/org/metaborg/spoofax/shell/client/Repl.java

```diff
@@ -47,6 +37,7 @@
         this.display = display;
         this.invoker = invoker;
 
+        // TODO: this does not really belong here.
```

Comment:
yea I agree. In fact we are still not sure whether `IEditor` should have a `setPrompt` and `setContinuationPrompt` method.